src/tasks/netcore/NetCoreTaskHelper.ts

```diff
@@ -223,36 +219,23 @@ export class NetCoreTaskHelper implements TaskHelper {
                 permissions: 'ro'
             };
 
-            const nugetVolume: DockerContainerVolume = {
+            const nugetRootVolume: DockerContainerVolume = {
                 localPath: path.join(os.homedir(), '.nuget', 'packages'),
                 containerPath: runOptions.os === 'Windows' ? 'C:\\.nuget\\packages' : '/root/.nuget/packages',
                 permissions: 'ro'
             };
 
-            let programFilesEnvironmentVariable: string | undefined;
-
-            if (isWindows()) {
-                programFilesEnvironmentVariable = process.env.ProgramFiles;
-
-                if (programFilesEnvironmentVariable === undefined) {
-                    throw new Error(localize('vscode-docker.tasks.netCore.programFilesUndefined', 'The environment variable \'ProgramFiles\' is not defined. This variable is used to locate the NuGet fallback folder.'));
-                }
-            }
-
-            const nugetFallbackVolume: DockerContainerVolume = {
-                localPath: isWindows() ? path.join(programFilesEnvironmentVariable, 'dotnet', 'sdk', 'NuGetFallbackFolder') :
-                    (isMac() ? MacNuGetPackageFallbackFolderPath : LinuxNuGetPackageFallbackFolderPath),
-                containerPath: runOptions.os === 'Windows' ? 'C:\\.nuget\\fallbackpackages' : '/root/.nuget/fallbackpackages',
+            const nugetUserVolume: DockerContainerVolume = {

```

Comment:
I'm not totally sure this volume mapping is even necessary for Windows; I think `ContainerUser` can access `C:\.nuget\packages`?

Reply:
In fact it looks like maybe we should always have been doing this: https://docs.microsoft.com/en-us/nuget/reference/nuget-config-file